conanfile.py

```diff
@@ -17,14 +17,15 @@ class CrashpadConan(ConanFile):
     default_options = {"linktime_optimization": False}
     exports = [ "patches/*", "LICENSE.md" ]
     short_paths = True
+    generators = "compiler_args"
 
     _commit_id = "c7d1d2a1dd7cf2442cbb8aa8da7348fa01d54182"
     _source_dir = "crashpad"
     _build_name = "out/Conan"
     _build_dir = os.path.join(_source_dir, _build_name)
 
     def build_requirements(self):
-        self.build_requires("depot_tools/cci.20201009")
+        self.build_requires("depot_tools_installer/20200515@bincrafters/stable")
```

Comment:
Can you please remind me why this change ?

Reply:
https://github.com/bincrafters/conan-crashpad/pull/13#issuecomment-802817025 and it's also in the commit message of https://github.com/bincrafters/conan-crashpad/pull/13/commits/74fb528e0237ad83c076a9a7999b408c251303b9.

Reply:
this looks like https://bugs.chromium.org/p/chromium/issues/detail?id=1067590#c7
It's probably worth reporting the issue on https://github.com/conan-io/conan-center-index/